src/main/kotlin/com/devfloor/untitled/comment/application/CommentService.kt

```diff
@@ -33,32 +34,24 @@ class CommentService(
     }
 
     @Transactional
-    fun create(
-        articleId: Long,
-        author: User,
-        request: CommentRequest,
-    ): CommentResponse {
-        val article = articleRepository.findByIdOrNull(articleId)
-            ?: throw EntityNotFoundException("존재하지 않는 게시글 입니다.")
+    fun create(author: User, request: CommentRequest): CommentResponse {
+        val article = articleRepository.findByIdOrNull(request.articleId)
+            ?: EntityNotFoundException.notExistsId(Article::class, request.articleId)
 
         return Comment(article, author, request.anonymous, request.content)
             .let(commentRepository::save)
             .let(::CommentResponse)
     }
 
-    private fun showByCommentId(commentId: Long): Comment =
-        commentRepository.findByIdOrNull(commentId)
-            ?: throw EntityNotFoundException("존재하지 않는 댓글입니다.")
-
-    @Transactional
     fun modifyByCommentId(commentId: Long, request: CommentModifyRequest) {
-        showByCommentId(commentId).modifyContent(request.content)
+        commentRepository.findByIdOrNull(commentId)
+            ?.modifyContent(request.content)
+            ?: EntityNotFoundException.notExistsId(Comment::class, commentId)
     }
 
-    fun destroyByCommentId(commentId: Long) {
-        showByCommentId(commentId).let {
-            commentFavoriteRepository.deleteAllByComment(it)
-        }
-        commentRepository.deleteById(commentId)
-    }
+    @Transactional
+    fun destroyByCommentId(commentId: Long) = commentRepository.findByIdOrNull(commentId)
+        ?.also(commentFavoriteRepository::deleteAllByComment)
+        ?.let(commentRepository::delete)
+        ?: EntityNotFoundException.notExistsId(Comment::class, commentId)
```

Comment:
여기 비교해보면 scope function때문에 너무 가독성이 떨어지는느낌이지 않아??
```suggestion
    fun destroyByCommentId(commentId: Long) {
        val comment = commentRepository.findByIdOrNull(commentId)
            ?: EntityNotFoundException.notExistsId(Comment::class, commentId)
        
        commentFavoriteRepository.deleteAllByComment(comment)
        commentRepository.delete(comment)
    }
```

Reply:
아 좀 그런가? 근데 아래 코드도 가독성 좋은 것 같아서 아래 스타일로 바꿔들게!!